### ISSUE TITLE
Display more that 5 e-people or groups per page

### ISSUE THREAD

**Issue by alawvt** [DSpace/dspace-angular#1764]
**Is your feature request related to a problem? Please describe.**
Display more that 5 e-people or groups per page. For common names it's helpful to see more choices simultaneously.

**Describe the solution you'd like**
DSpace 6 shows 15 per page. I20 might be better.

**Describe alternatives or workarounds you've considered**

**Additional context**

**Comment by mwoodiupui** [comment]
The sizes of a number of list displays should be configurable.

**Comment by alawvt** [comment]
@mwoodiupui, is that "should" mean "they should be configurable but they are not currently" or "they should already be configurable"? Thanks.

**Comment by mwoodiupui** [comment]
@alawvt Sorry, my comment was unclear.  The sizes of a number of list displays should be **made** configurable.  This issue is one example.  Maybe we should all put our heads together and make a list.  Locally I've been asked to resize the list of results from a search, and that too is hard-coded, at least in 7.3.

**Comment by alawvt** [comment]
@mwoodiupui, thanks for the clarification. I would be happy to help make such a list. I'm still testing my way through DSpace 7 Demo.

**Comment by tdonohue** [comment]
Agreed, this should be configurable.  It appears this is hardcoded in two locations:

* For EPeople registry listing, the page size is hardcoded here: https://github.com/DSpace/dspace-angular/blob/main/src/app/access-control/epeople-registry/epeople-registry.component.ts#L62
* For Group registry listing, the page size is hardcoded here: https://github.com/DSpace/dspace-angular/blob/main/src/app/access-control/group-registry/groups-registry.component.ts#L57

Both could easily be made configurable in a manner similar to #1771 .

Moving this over to the 7.4 board.  For now, I'm flagging this as medium priority as this page only is usable to full admins (so few are impacted), but it might be fixed alongside other similar issues like #1749 and #1750
